### DISCUSSION TITLE
Welcome to DSA5maptool Discussions!

### DISCUSSION THREAD

**Discussion by Lector** [Lector/DSA5maptool#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Naxos84** [comment]
Hi.

Cool, dass das so schnell mit den Diskussionen geklappt hat.
Nochmal vielen Dank, dass du dir die Mühe gemacht hast diese Kampagne zu schreiben. Da steckt exht mega viel Arbeit drin, wenn man sich das mal im Detail anschaut.

Ich selber bin noch recht neu in MapTool (und auch in DSA) unterwegs und muss mich an vielen Stellen noch umschauen/rein lesen, aber als Software Entwickler sollte mir das zumindest MapTool seitig schnell gelingen.

Mich interessiert natürlich ganz stark, wie wir das ganze evtl. noch verbessern können.
Dazu zählt Vereinfachung der Makros und vielleicht (sofern möglich) auch Performance Verbesserung durch Auslagern von Assets (sofern das geht).

Ich bin gespannt ob und in wie fern ich ggf. helfen kann.

**Comment by Lector** [comment]
Dann erstmal herzlich Willkommen hier :)

Falls zu zu irgendetwas Frage hast kannst du dich gerne melden.

Auch für Verbesserungen bin ich immer offen. Aktuell gibt es inhaltlich keine konkrete Baustelle die offen ist. Wenn du also hier irgendwelche Wünsche oder Vorschläge hast kannst du die gerne hören lassen.
